src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -53,12 +76,34 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
           class="markdown ${classMap({
             "no-header": !this._config.title,
           })}"
-          .content="${this._config.content}"
+          .content="${this._content}"
         ></ha-markdown>
       </ha-card>
     `;
   }
 
+  private async _connect() {
+    if (!this._unsubRenderTemplate && this._hass && this._config) {
+      this._unsubRenderTemplate = await subscribeRenderTemplate(
```

Comment:
Another thing, I would not do this with an `await` but instead store the promise in `this._unsubRenderTemplate`. And instead of `try…catch` add a `catch` to the promise. That way connecting and then disconnecting an element before the subscribe function returned will work. 

Inside disconnect you do then `this._unsubRenderTemplate.then(unsub => { unsub(); this._unsubRenderTemplate = undefined; })`

Reply:
This was a pain to get all the exceptions...

I've tested it agains the dev branch of the backend. Seems to work with the fallback and nonexisting connections and stuff now.